Try installing ifort on windows

scharlton2 opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

scharlton2 commented:
COMPONENTS

scharlton2 commented:
@conda-forge-admin, please rerender

scharlton2 commented:
@conda-forge-admin, please rerender

jdhughes-dev commented on recipe/meta.yaml:
```diff
@@ -10,13 +10,11 @@ source:
-  number: 0
-  skip: true  # [win]
+  number: 1
+  skip: true  # [not win]
```
Does this need to be modified to build non-windows versions as well. I only see the Azure windows build.

scharlton2 replied:
Yes, I'm just testing the windows part right now.  (I didn't want to waste a bunch of resources on the other builds).  @jdhughes-usgs do you know if the meson build for intel fortran is statically linked, or does it need the intel fortran runtime dlls?

jdhughes-dev replied:
The meson build will be statically linked

scharlton2 commented:
@conda-forge-admin, please rerender

scharlton2 commented:
@conda-forge-admin, please rerender

scharlton2 commented:
Just searched the conda-forge groups and found a feedstock doing the same thing for building on windows:

https://github.com/conda-forge/openfast-feedstock

https://github.com/conda-forge/openfast-feedstock/blob/main/recipe/bld.bat

So, I'm much less hesitant to merge this now.  What do you think @jdhughes-usgs?  You might want to look at the changes I made to the meson options. (I changed debug to false and removed the optimization=0 setting)

jdhughes-dev commented:
> Just searched the conda-forge groups and found a feedstock doing the same thing for building on windows:
> 
> https://github.com/conda-forge/openfast-feedstock
> 
> https://github.com/conda-forge/openfast-feedstock/blob/main/recipe/bld.bat
> 
> So, I'm much less hesitant to merge this now. What do you think @jdhughes-usgs? You might want to look at the changes I made to the meson options. (I changed debug to false and removed the optimization=0 setting)

@scharlton2 The meson changes are good - they are consistent with how we compile on CI. I think we should merge the PR.

scharlton2 commented:
@jdhughes-usgs I think you're going to have to merge it.  I don't think I have the power yet.
